xbmc/utils/URIUtils.cpp

```diff
@@ -936,8 +936,10 @@ bool URIUtils::IsHTTP(const std::string& strFile)
   if (HasParentInHostname(url))
     return IsHTTP(url.GetHostName());
 
-  return IsProtocol(strFile, "http") ||
-         IsProtocol(strFile, "https");
+  std::string strProtocol = (bTranslate ? url.GetTranslatedProtocol() : url.GetProtocol());
+
+  return (strProtocol == "http" ||
+         strProtocol ==  "https";
```

Comment:
```suggestion
         strProtocol ==  "https");
```
missing close previously opened parentheses 😃

Reply:
Well, seeing code suggested for merge that does not even compile makes me a bit worried about how this got tested.

Reply:
Well I said I wasn't ready yet, that's why ;-) Obviously I will test it once again, when done.

Reply:
> Well I said I wasn't ready yet, that's why ;-)

In this case, please add the [WIP] label to this PR. So, nobody will waiste time with reviewing in-progress stuff.

Reply:
Yes, obviously this wasn't my initial intention. This started as a simple fix and later on I decided to cleanup/refactor some related stuff....

Please note I just committed  the final changes so it's ready for its last review.